cns/grpc/proto/server.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
```

Comment:
Like we discussed, we want to put all `protoc'-generated Go code in a different directory, so let's remove this file